chapters/solutions.Rmd

```diff
@@ -1582,7 +1654,53 @@ most items would be checked off of the list.
 
 ### Exercise \@ref(packaging-ex-data) {-}
 
-FIXME
+The central idea is that a Python file can find out what directory it was read from,
+so it can look for other files in that directory.
+This means that we don't have assume the package has been installed in a particular place
+or that some environment variable will tell us where to look for our data.
+To test this idea out:
+
+1.  Create a directory called `data-package`.
+
+1.  Put a copy of `collated.csv` in that directory.
+
+1.  Add a file called `collated.py` to that directory.
```

Comment:
Is this confusing because we already have a script we've been working on called `collate.py`?

Maybe it would be better if this data package contained the word count csv files for a number of our books (e.g. Dracula, Jane Eyre, etc) rather than the collated count?